lib/watir/elements/element.rb

```diff
@@ -576,7 +576,8 @@ def element_class_name
     end
 
     def selector_string
-      @selector.inspect
+      return @selector.inspect if @parent.is_a? Browser
+       "#{@parent.send :selector_string} --> #{@selector.inspect}" 
```

Comment:
make selector_string protected, not private and just call `@parent.selector_string`